releasenotes/2021-February-3.md

```diff
@@ -0,0 +1,192 @@
+This release notes contain new changes for ACS Calling Web (JavaScript) SDK v1.0.0-beta.4
+
+# Bug fixes
+1. Fix CallEndReason.subCode lower case 'c' typo 
+2. Fix CallAgent.join() api accepting undefined CallContext. Api must now take a GroupLocator or MeetingLocator 
+
+# Breaking API Changes
+1. New flow for collection subscriptions:
```

Comment:
nit, do mention that application has to 
1) inspect existing values of properties & collections 
2) subscribe to property & collection update events

In order to get current & future values